code/game/objects/items/spear.dm

```diff
@@ -139,7 +139,9 @@
 	if(iseffect(AM)) //and no accidentally wasting your moment of glory on graffiti
 		return
 	user.say("[war_cry]", forced="spear warcry")
-	explosive.forceMove(AM)
+	if(ismob(AM))
+		user.forceMove(get_turf(AM))
```

Comment:
Make this do a normal move so it still factors in any immobilizers you might have, walls, etc. It's fine if it fails.

Reply:
that won't work cuz it'll proc bump code and make the two people swap places

Reply:
Make 'em rest or something beforehand then, but either way we shouldn't be forceMove'ing this.

Reply:
![image](https://user-images.githubusercontent.com/35135081/155420482-52c82347-5f9c-4635-abf7-1319abc7bff9.png)

I'm fine with this answer

Reply:
Also even with Goof's answer still make them rest or something so they don't swap in place